### DISCUSSION TITLE
articles/cycling-in-shanghai-after-lockdown

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#171] [Comments]
# articles/cycling-in-shanghai-after-lockdown

上海解封后，端午放假第一天，我带着七岁的儿子大卫早早起来去骑行。

https://hutusi.com/articles/cycling-in-shanghai-after-lockdown

**Comment by bmxklYzj** [comment]
>2019年及之前的年份已是不可再回，那个自在的世界也不可再得。

**Reply by hutusi** [reply]
时间不可倒退，但还是希望世界能变得更好~

**Comment by parkergrt** [comment]
照片用什么设备拍的呀，看着很不错

**Reply by hutusi** [reply]
就用手机拍摄的~
